README.md

```diff
@@ -1,11 +1,13 @@
-# pinecone-client
+# pinecone-python-client
```

Comment:
This was the only thing I changed in here and it seems like there was some auto-formatting applied.